data/human/news.txt

```diff
@@ -739,6 +791,28 @@ news "human mechanic"
 
 
 
+news "human ship salesperson"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Used Spaceship Salesman"
+			"Used Spaceship Saleswoman"
+	message
+		word
+			`"`
+		word
+			"When one hatch closes, another one opens. Other than that it's a pretty good spaceship."
+			"This is a very durable spaceship. By the way; do want to buy extended warranty to cover all the breakdowns this ship typically experiences in the first year?"
```

Comment:
I know, but I actually experienced this when I was buying a washing machine; after I said I'll buy it, then the salesman started telling me everything that could break-down because he wanted to also sell me an extended warranty.

I'll replace the text by something similar.